### DISCUSSION TITLE
doesn't React support the feature [SFC custom block for Route Data ]?

### DISCUSSION THREAD

**Discussion by bxzhou2008** [hannoeru/vite-plugin-pages#449] [Ideas]
### Describe the bug

is there any way to define route data in react

### Reproduction

no repo

### System Info

```shell
normally env
```

### Used Package Manager

pnpm

### Validations

- [X] Follow the [Code of Conduct](https://github.com/hannoeru/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the docs.
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A please open a GitHub Discussion.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by vaclock** [comment]
so nobody need this feature? i think it is an important feature
